homeassistant/components/device_tracker/netgear.py

```diff
@@ -12,21 +12,29 @@
 from homeassistant.components.device_tracker import (
     DOMAIN, PLATFORM_SCHEMA, DeviceScanner)
 from homeassistant.const import (
-    CONF_HOST, CONF_PASSWORD, CONF_USERNAME, CONF_PORT)
+    CONF_HOST, CONF_PASSWORD, CONF_USERNAME, CONF_PORT, CONF_URL,
+    CONF_DEVICES, CONF_EXCLUDE)
 
-REQUIREMENTS = ['pynetgear==0.3.3']
+REQUIREMENTS = ['pynetgear==0.3.4']
 
 _LOGGER = logging.getLogger(__name__)
 
-DEFAULT_HOST = 'routerlogin.net'
+CONF_APS = 'accesspoints'
+
 DEFAULT_USER = 'admin'
-DEFAULT_PORT = 5000
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Optional(CONF_HOST, default=DEFAULT_HOST): cv.string,
+    vol.Optional(CONF_HOST, default=''): cv.string,
     vol.Optional(CONF_USERNAME, default=DEFAULT_USER): cv.string,
     vol.Required(CONF_PASSWORD): cv.string,
-    vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port
+    vol.Optional(CONF_PORT, default=None): vol.Any(None, cv.port),
+    vol.Optional(CONF_URL, default=''): cv.string,
```

Comment:
I would suggest that instead of using `url:` we should go with `ssl:` as a lot of other platforms are doing this already. Also, this would make the change no-breaking as it's an optional new feature.

Reply:
The change is already not breaking :) If no URL the rest is used.
Anyway I can go with SSL instead.